src/dashboard/src/fpr/views.py

```diff
@@ -375,17 +374,13 @@ def idcommand_delete(request, uuid):
         if 'disable' in request.POST:
             command.enabled = False
             messages.info(request, _('Disabled.'))
-            for obj in dependent_objects:
-                obj['value'].enabled = False
-                obj['value'].save()
```

Comment:
This was disabling `IDRule` objects when the associated `IDCommand` was disabled.